mathml/presentation-markup/mpadded/mpadded-002.html

```diff
@@ -59,7 +59,7 @@
       for (var i = 0; i <= 2; i++) {
         var mpadded = getBox("mpadded" + i);
         assert_approx_equals(mpadded.width, 25*(1+i%3), epsilon, "width" + i);
-        assert_approx_equals(getBox("baseline").bottom - mpadded.top, 25*(1+(i+1)%3), epsilon, "height" + i);
+        assert_approx_equals(getBox("baseline2").bottom - mpadded.top, 25*(1+(i+1)%3), epsilon, "height" + i);
         assert_approx_equals(mpadded.bottom - getBox("baseline").bottom, 25*(1+(i+2)%3), epsilon, "depth" + i);
```

Comment:
mmh this one is still baseline? (I didn't read the test again, but please check carefully)